src/BinaryParsers/PEBinary/ProgramDatabase/Pdb.cs

```diff
@@ -134,6 +143,42 @@ private void WindowsNativeLoadPdbUsingDia(string pePath, string symbolPath, stri
 
         public bool IsStripped => this.GlobalScope.IsStripped;
 
+        public PdbFileType FileType

```

Comment:
You should cache this property value rather than computing it every time.

Reply:
We are caching unless it's unknown

---
In reply to: [563214200](https://github.com/microsoft/binskim/pull/335#discussion_r563214200) [](ancestors = 563214200)